### DISCUSSION TITLE
Image cannot be centered in Hexo blog.

### DISCUSSION THREAD

**Discussion by SightVanish** [hexojs/hexo#5021] [Question / Answer] [answered]
My image can be displayed in hexo but cannot be centered.
Here is my code.
```
<div align="center">
    <img src="./folder/image.jpg"></img>
</div>
```
However, instead of image, source code is displayed.
What is the cause to this problem? How can I center image?

**Comment by renbaoshuo** [comment, marked as answer]
```html
<p align="center"><img src="./image.png"></p>
```

This works for me.

**Reply by SightVanish** [reply]
WOW, it works for me. Thank you!
